<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnKameraAc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Kamera</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCam">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>601</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAnimation">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Dönen Zebra</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnYuzTanima">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Yüz tanıma</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
